src/libs/Environment/Environment.js

```diff
@@ -31,8 +37,19 @@ function getEnvironmentURL() {
     });
 }
 
+/**
+ * Get the corresponding oldDot URL based on the environment we are in
+ *
+ * @returns {Promise<string>}
+ */
+function getOldDotEnvironmentURL() {
+    return getEnvironment()
```

Comment:
Using `getEnvironment` will break olddot urls for those who don't use `.env` file. The olddot url will fallback to the dev url which is not accessible.

Reply:
But how one can not have .env file. I think you meant contributors.

Reply:
Yes, external contributors are setup so that they don't need an `.env` file, even if they have one, they won't be able to access `https://www.expensify.com.dev` 

Is it OK if this functionality doesn't work for external contributors, when they are using the DEV build? 

Since external contributors are using the PROD API (In DEV via proxy), perhaps this link should point to https://www.expensify.com for external contributors during DEV

Reply:
Got it,
So technically this doesn't sound like a bug.

As @s77rt, We can ask contributors to make it work by overriding the URL in CONST or updating `getOldDotEnvironmentURL()`

Not sure, if we should add it to README

Reply:
I think we can post in slack and give a heads up to contributors

Reply:
Yeah README is an overkill for such minor detail that contributors may not even have to deal with. Let's do nothing for now, if we get reports on this we may consider adding it to README then.

Reply:
Sounds good! Thank you!

Reply:
Posting this on Slack seems as a good idea, let's do it.